packages/elements/src/overlay/managers/interaction-lock-manager.ts

```diff
@@ -162,12 +162,12 @@ export class ScrollLockManager {
    */
   private saveScrollPosition(): void {
     if (document.scrollingElement) {
-      this.scrollTop = document.scrollingElement.scrollTop;
-      this.scrollLeft = document.scrollingElement.scrollLeft;
+      this.scrollTop = document.scrollingElement?.scrollTop || 0;
```

Comment:
`scrollingElement` could be `HTMLBodyElement` in quirk mode only. I don't think we need to complicate ourselves with this ancient feature.

Reply:
Agreed and reverted.